### ISSUE TITLE
package_data/static/dat partitioned by template type

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#582]
The current structure is flat and assumes file suffixes are enough to know which files to copy, e.g.

```python
    for f in glob.glob(str(_STATIC_FOLDER.join('dat', '*.sdds'))):
```

Rather there should be a subdirectory (e.g. srw or elegant) in package_data/static/dat that holds the files per template type

**Comment by robnagler** [comment]
Fix ebd7576

**Comment by robnagler** [comment]
NOTE: You need latest pykern.
